plugin.video.youtube/resources/lib/youtube/helper/yt_specials.py

```diff
@@ -51,17 +56,22 @@ def _process_browse_channels(provider, context, re_match):
     guide_id = context.get_param('guide_id', '')
     if guide_id:
         json_data = provider.get_client(context).get_guide_category(guide_id)
+        if not v3.handle_error(provider, context, json_data):
+            return False
         result.extend(v3.response_to_items(provider, context, json_data))
         pass
     else:
         json_data = provider.get_client(context).get_guide_categories()
+        if not v3.handle_error(provider, context, json_data):
+            return False
         result.extend(v3.response_to_items(provider, context, json_data))
         pass
 
     return result
 
 
 def _process_disliked_videos(provider, context, re_match):
+    provider.set_content_type(context, kodion.constants.content_type.EPISODES)
```

Comment:
Based on your request, and reading http://i.imgur.com/TeXMOMx.png from (https://kodi.tv/kodi-v17-krypton-default-skin-next-gen/) I have removed the override views and set them all to videos. It is limited in view selection especially on < Krypton so I expect this will not be something users are going to like.